docs/src/main/asciidoc/using-vertx.adoc

```diff
@@ -0,0 +1,483 @@
+include::./attributes.adoc[]
+= {project-name} - Using Eclipse Vert.x
+
+Eclipse https://vertx.io[Vert.x] is a toolkit for building reactive applications.
+It is designed to be lightweight and embeddable.
+Vert.x defines a reactive execution model and provides a large ecosystem.
+Quarkus integrates Vert.x to implement different forms of reactive.
```

Comment:
Oh yeah? "different forms of reactive"? Any specific reactive thing or all things reactive? I think there's a word missing ;)